From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Joel Stanley <joel@jms.id.au>
Date: Wed, 14 Dec 2016 13:20:09 +1030
Subject: [PATCH 12/12] Release 4.4.38-openpower1

 - Backports to fix building with modern binutils
   * powerpc: Convert cmp to cmpd in idle enter sequence
   * powerpc/boot: Request no dynamic linker for boot wrapper

 - Add fix for our xhci workarounds. This fixes a hang in Petitboot when
   shutting down with a usb device attached to an external usb hub.
    * usb/xhci: allow xhci_check_bandwith in XHCI_STATE_REMOVING

 - Move to latest upstream stable release
   * Includes fixes from 4.4.36, 4.4.37, 4.4.38

Signed-off-by: Joel Stanley <joel@jms.id.au>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 6876efe0d735..fd471c6e111c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 4
 PATCHLEVEL = 4
 SUBLEVEL = 38
-EXTRAVERSION =
+EXTRAVERSION = -openpower1
 NAME = Blurry Fish Butt
 
 # *DOCUMENTATION*
-- 
2.11.0

